db/dbformat.h

```diff
@@ -168,48 +168,92 @@ inline void UnPackSequenceAndType(uint64_t packed, uint64_t* seq,
 EntryType GetEntryType(ValueType value_type);
 
 // Append the serialization of "key" to *result.
+//
+// input [internal key]: user_key | seqno + type
+// output before:
```

Comment:
I am a little confused about what "output before:" means in the comments.

Reply:
It was trying to mean that the output argument should be empty before the call. Thank you for calling out that it is confusing. I have updated it to say empty specifically, and added some brackets to mark the actual content in the input and output fields so that there is some contrast.

Reply:
I see. I did not initially realize that output means the `result` parameter.